--- swm-head/swm_param.py	2023-03-20 16:37:04.169002175 +0000
+++ swm-master/swm_param.py	2023-03-20 15:59:50.925845141 +0000
@@ -7,8 +7,8 @@
     param = dict()

     ## parameters
-    param['nx'] = 256               # number of grid points in x-direction
-    param['ny'] = 256               # number of grid points in y-direction
+    param['nx'] = 128 * 4               # number of grid points in x-direction
+    param['ny'] = 128 * 4               # number of grid points in y-direction

     param['Lx'] = 3840e3            # basin width L [meters]
     param['Ly'] = 3840e3            # north-south basin extent [meters]
@@ -17,7 +17,7 @@
     param['H'] = 500.               # water depth [m]

     param['cfl'] = .9               # desired CFL-criterion, recommended 0.9, uses RK4
-    param['Ndays'] = 5*365             # number of days to integrate
+    param['Ndays'] = 200             # number of days to integrate

     param['dat_type'] = np.float32  # single/double precision use np.float32 or np.float64

@@ -38,7 +38,7 @@
     # OUTPUT - of netcdf4, info_txt, parameters and scripts
     param['output'] = 1             # or 0 for no data storage
     param['output_dt'] = 6*3600    # every hours*3600 therefore in seconds
-    param['outputpath'] = '/network/aopp/cirrus/pred/kloewer/swm_bf_cntrl/' # sets the path for netcdf output, else choose ''
+    param['outputpath'] = 'output/' # sets the path for netcdf output, else choose ''

     ## SET UP derived parameters
     set_grid()
